recipes/px4tools/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "px4tools" %}
+{% set version = "0.3.7" %}
+{% set sha256 = "165351d235869e29449003c1bd4b36c64561f85e1932f0a12e739a25dbfc8227" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/dronecrew/px4tools/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+
+  # noarch_python: True
+  # preserve_egg_dir: True
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - px4tools = px4tools:main
+    #
+    # Would create an entry point called px4tools that calls px4tools.main()
+
+    - px42csv=px4tools.px42csv:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scipy
+    - numpy
+    - pandas
+    - matplotlib
+    - basemap
```

Comment:
Please drop everything that is not needed at build time. Namely `python`, `setuptools`, and anything in the `setup_requires` section.